serverless/messaging/api-cli/python-node-sns.mdx

```diff
@@ -91,13 +91,7 @@ for i := 0; i < 10; i++ {
 
 #### Subscribe to an SQS queue from the same Scaleway Project
 
-```go
-_, _ = awsSns.Subscribe(&sns.SubscribeInput{
-  Endpoint: aws.String(QueueArn),
-  Protocol: aws.String("sqs"),
-  TopicArn: createTopicResponse.TopicArn,
-})
-```
+This is no longer supported. Watch this space for future developments.
```

Comment:
Just wondering if we should say `no longer supported` or `not yet supported`. The 2nd option looks more appropriate for people reading this doc for the first time and we know we don't have a lot of SNS users today.

Same comment applies to SNS Dead Letter Queues.